apps/vr-tests-react-components/src/stories/RatingDisplay.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { CircleRegular, CircleFilled, SquareRegular, SquareFilled } from '@fluentui/react-icons';
+import { RatingDisplay } from '@fluentui/react-rating-preview';
+import { TestWrapperDecoratorFixedWidth } from '../utilities/TestWrapperDecorator';
+
+storiesOf('RatingDisplay Converged Size', module)
```

Comment:
A few stories to add:
* No value: `<RatingDisplay />`
* Rounded up: `<RatingDisplay value={3.8} />`
* Rounded down: `<RatingDisplay value={3.7} />`